README.md

```diff
@@ -1 +1,53 @@
-# MyPortfolio
\ No newline at end of file
+# Portfolio
+
+> My Personal Portfolio
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+**Run HTML file**
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+**pass Linters Tests**
+
+### Deployment
+
+## Authors
+
+👤 **Kareem Magdy Wilson**
+
+- GitHub: [KareemWilson](https://github.com/KareemWilson)
+- LinkedIn: [Kareem Wilson](https://www.linkedin.com/in/kareem-wilson/)
+- Instagram [CodeAfternoon](https://www.instagram.com/code.afternoon/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+
+## 📝 License
+
+This project is [Kareem wilson](https://github.com/KareemWilson) licensed.
```

Comment:
- [ ] Well done for adding the License section 👏 However, kindly use the MIT license